pkg/local_object_storage/engine/inhume.go

```diff
@@ -324,7 +323,7 @@ func (e *StorageEngine) isLocked(addr oid.Address) (bool, error) {
 	return locked, outErr
 }
 
-func (e *StorageEngine) processExpiredObjects(_ context.Context, addrs []oid.Address) {
```

Comment:
tbh, i am not sure it _should never be used there_. expired object handling _can_ be interrupted, and an expired now object will still be expired in every next epoch, so if we are about to stop working, such operation is safe to stop immediately. however, none of the current "iteration" storage engine operations supports context so ok